NapDatabaseExport/MSSQLServerProvider.cs

```diff
@@ -112,7 +129,7 @@ public override bool TryConnect ()
         {
             var tables = new List<string> ();
 
-            using (var dr = ExecuteReader ("SELECT name FROM sys.databases"))
+            using (var dr = ExecuteReader ("SELECT name FROM sys.databases WHERE database_id > 4 ORDER BY name"))
```

Comment:
Why filter by database_id?

Reply:
There are 4 system database: `master`, `model`, `msdb` and `tempdb`. (Actually 5 with `publish` replication database but this cannot be filtered by id). You don't want NRA asking irrelevant questions like "What is master database? Do you store something there?" :-))

Reply:
OK